src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1169,8 +1180,48 @@ void uct_rc_mlx5_iface_common_event_cq(uct_ib_iface_t *ib_iface,
     iface->cq[dir].cq_sn++;
 }
 
+static UCS_F_ALWAYS_INLINE void
+uct_rc_mlx5_iface_emplace_compressed_cqes(uct_ib_mlx5_cq_t *cq,
+                                          unsigned cqe_title_index, int is_rx)
+{
+    uct_ib_mlx5_cq_decomp_t *cq_decomp = &cq->cq_decomp;
+    struct mlx5_cqe64 *title = uct_ib_mlx5_get_cqe(cq, cqe_title_index);
+    struct mlx5_cqe64 *mini_cqe_arr;
+    struct mlx5_cqe64 *cqe;
+    uct_ib_mlx5_mini_cqe8_t *mini_cqe;
+    int cqe_pos, next_arr_index = cqe_title_index + 1;
+
+    /* Initialize the decompression related stuff */
+    cq_decomp->block_size   = ntohl(title->byte_cnt);
+    cq_decomp->left         = cq_decomp->block_size;
+    cq_decomp->wqe_counter  = ntohs(title->wqe_counter);
+    cq_decomp->mini_arr_idx = 0;
+    memcpy(&cq_decomp->title, title, sizeof(struct mlx5_cqe64));
```

Comment:
why need to copy?

Reply:
Discussed orally, pls reopen in case of questions